packages/jest/rules-snapshot.json

```diff
@@ -40,18 +40,16 @@
   "jest/no-standalone-expect": "error",
   "jest/no-test-prefixes": "error",
   "jest/no-test-return-statement": "error",
-  "jest/no-try-expect": "error",
   "jest/prefer-hooks-on-top": "error",
   "jest/prefer-spy-on": "error",
   "jest/prefer-strict-equal": "error",
-  "jest/prefer-to-be-null": "error",
-  "jest/prefer-to-be-undefined": "error",
+  "jest/prefer-to-be": "error",
```

Comment:
Supersedes `to-be-x`: https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-to-be.md

This may actually be breaking, because it looks like it will make us use `toBe` for all primitives that don't have specific `toBeX` matchers.